packages/ui-extensions/docs/surfaces/point-of-sale/staticPages/pages/versions.doc.ts

```diff
@@ -30,6 +30,7 @@ const data: LandingTemplateSchema = {
 - Added support for the ${TargetLink.PosPurchasePostBlockRender} target.
 - Added support for the ${TargetLink.PosOrderDetailsBlockRender} target.
 - Added support for the ${TargetLink.PosCustomerDetailsBlockRender} target.
+- Deprecated the [ActionItem component](/docs/api/pos-ui-extensions/components/ActionItem). Please use the [Button component](/docs/api/pos-ui-extensions/components/Button) instead.
```

Comment:
Did we decide when we are actually removing the `ActionItem`? Is having one version enough of a buffer for partners to migrate?